master/buildbot/statistics/capture.py

```diff
@@ -185,24 +205,35 @@ class CaptureData(Capture):
     """
 
     def __init__(self, data_name, builder_name, callback=None):
-        self.data_name = data_name
-        self.builder_name = builder_name
+        self._data_name = data_name
+        self._builder_name = builder_name
+
+        def identity(x):
+            return x
 
         if not callback:
-            callback = lambda x: x
+            callback = identity
 
         routingKey = ("stats-yieldMetricsValue", "stats-yield-data")
```

Comment:
I understand that, however without that documentation it's a bit difficult to understand what happens here.